docs/source/commandline/commandline-all.rst

```diff
@@ -947,6 +964,88 @@ Core Vector Options
 
 
 
+Core Video OpenGL Debugging Options
+-----------------------------------
+
+These 4 options are for compatibility in **-video opengl**.  If you report rendering artifacts you may be asked to try messing with them by the devs, but normally they should be left at their defaults which results in the best possible video performance.	
+
+.. _mame-commandline-glforcepow2texture:
+
+**-[no]gl_forcepow2texture**
+
+	Always use only power-of-2 sized textures (default *off*)
+
+.. _mame-commandline-glnotexturerect:
+
+**-[no]gl_notexturerect**
+
+	Don't use OpenGL GL_ARB_texture_rectangle (default *on*)
+
+.. _mame-commandline-glvbo:
+
+**-[no]gl_vbo**
+
+    Enable OpenGL VBO,  if available (default *on*)
+
+.. _mame-commandline-glpbo:
+
+**-[no]gl_pbo**
+
+    Enable OpenGL PBO,  if available (default *on*)
+
+
+Core Video OpenGL GLSL Options
+------------------------------
+
+
+.. _mame-commandline-glglsl:
+
+**-gl_glsl**
+
+	Enable OpenGL GLSL, if available (default *off*)
+
+.. _mame-commandline-glglslfilter:
+
+**-gl_glsl_filter**
```

Comment:
As of MAME 0.194 this also accepts 2 for bicubic.

Reply:
Fixed on my side.